### ISSUE TITLE
ScanRef.last_history() returns oldest, not newest history

### ISSUE THREAD

**Issue by McGarryBrendan** [tenable/Tenable.io-SDK-for-Python#74]
The documentation for the `tenable_io.helpers.scan.ScanRef.last_history()` is incorrect.

https://github.com/tenable/Tenable.io-SDK-for-Python/blob/master/tenable_io/helpers/scan.py#L448
> "Get last (most recent) scan history if exists."

The `last_history()` function actually returns the oldest, not the most recent history. 

The call to `histories()` returns an array of `tenable_io.api.models.ScanDetailsHistory` in descending order, with index 0 containing the most recent scan history.

Reproduction:
```
import datetime
from tenable_io.client import TenableIOClient

# init client, retrieve scan reference by id
c = TenableIOClient()
scanref = c.scan_helper.id(12)

# display last history information
lh = scanref.last_history()
print(lh.creation_date, lh.status)

# pull all histories and display info
for i, h in enumerate(scanref.histories()):
    time = datetime.datetime.fromtimestamp(int(h.creation_date)).strftime('%Y%m%d %H:%M:%S')
    print(i, h.creation_date, time, h.status)
```

Output:
```
1513365288 canceled
0 1515088627 20180104 12:57:07 completed
1 1514995187 20180103 10:59:47 completed
2 1514926583 20180102 15:56:23 completed
3 1514925168 20180102 15:32:48 completed
4 1514924662 20180102 15:24:22 canceled
5 1514908627 20180102 10:57:07 completed
6 1513894382 20171221 17:13:02 completed
7 1513892888 20171221 16:48:08 canceled
8 1513889266 20171221 15:47:46 completed
9 1513888910 20171221 15:41:50 canceled
10 1513885656 20171221 14:47:36 canceled
11 1513804701 20171220 16:18:21 completed
12 1513804590 20171220 16:16:30 canceled
13 1513804327 20171220 16:12:07 canceled
14 1513804088 20171220 16:08:08 canceled
15 1513803058 20171220 15:50:58 canceled
16 1513802921 20171220 15:48:41 canceled
17 1513802640 20171220 15:44:00 canceled
18 1513802440 20171220 15:40:40 canceled
19 1513802358 20171220 15:39:18 canceled
20 1513801832 20171220 15:30:32 canceled
21 1513800061 20171220 15:01:01 canceled
22 1513699898 20171219 11:11:38 completed
23 1513629207 20171218 15:33:27 completed
24 1513626110 20171218 14:41:50 completed
25 1513623447 20171218 13:57:27 completed
26 1513368667 20171215 15:11:07 completed
27 1513365288 20171215 14:14:48 canceled
```

This issue can be resolved by returning index 0:
https://github.com/tenable/Tenable.io-SDK-for-Python/blob/master/tenable_io/helpers/scan.py#L454
`return histories[-1] if len(histories) else None` \
should be changed to:
`return histories[0] if len(histories) else None`

If this function behavior is correct, then the function documentation just needs to be updated to reflect the behavior.
> "Get last (most recent) scan history if exists."
to
> "Get last (oldest) scan history if exists."

**Comment by tenableAndrew** [comment]
Thanks for finding this @McGarryBrendan, it looks like there was a change in expected behavior in the underlying public API. I'll get a fix up shortly.

**Comment by tenableAndrew** [comment]
Should be fixed in #75
